### ISSUE TITLE
Copy pasting does not work

### ISSUE THREAD

**Issue by UAce** [getinsomnia/insomnia#2398]
Every time I copy paste and send a request, Insomnia treats the pasted text as empty.

**Steps to reproduce:**
1. Copy a URL
2. Create a new request or modify an existing request
3. Paste the URL to the request
4. Send the request
5. Click on another request and come back to the newly created/modified request. Notice that the URL is blank
_**Note**: this behaviour also happens for request body, headers, etc. Also, if you type something after the text you copy pasted,  it would magically work._

**Expected behavior**
Copy pasting should work

**Screenshots**
Step 2.
![image](https://user-images.githubusercontent.com/19378441/87324151-b626fd80-c4fd-11ea-8988-7ce67e50abfe.png)

Step 4.
![image](https://user-images.githubusercontent.com/19378441/87324221-cb039100-c4fd-11ea-9f04-7a0bc99d5510.png)

Step 5.
![image](https://user-images.githubusercontent.com/19378441/87324330-ef5f6d80-c4fd-11ea-8cf3-478dba1dc6bd.png)

**Context**
OS: `Ubuntu 18.04.3 LTS`
Insomnia version:
![image](https://user-images.githubusercontent.com/19378441/87324481-22a1fc80-c4fe-11ea-88c1-af056daed468.png)

**Comment by Necrontyrr** [comment]
Hiya, this has been resolved in the latest release - 2020.3.3. Please update your client. 😊

**Comment by ajc308** [comment]
**Update**: I posted this as the previous comment was being added. Confirmed this is now working in 2020.3.3 for me. Thank you!

---

I am experiencing the same error after upgrading to Insomnia 2020.3.0 (macOS Catalina 10.15.4)

Recording of copying and pasting values into headers and seeing them disappear:

![Insomnia_copy_paste_issue](https://user-images.githubusercontent.com/7770139/87330234-45381380-c506-11ea-95e7-9e61ef80d65a.gif)
